operations/experimental/enable-nfs-broker-backup.yml

```diff
@@ -0,0 +1,14 @@
+---
+  - type: replace
+    path: /instance_groups/name=backup-restore/jobs/-
+    value:
+      name: nfsbroker-bbr
+      release: nfs-volume
+      properties:
+        nfsbroker:
+          db_driver: ((nfsbroker-db-driver))
+          db_username: ((nfsbroker-db-username))
+          db_password: ((nfsbroker-db-password))
+          db_hostname: ((nfsbroker-db-hostname))
+          db_port: ((nfsbroker-db-port))
+          db_name: ((nfsbroker-db-name))
```

Comment:
... as are port and db name: https://github.com/cloudfoundry/cf-deployment/blob/master/operations/community/enable-volume-service-nfs.yml#L74-L75